LeagueOfLegends/BaseGameModule.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Games.LeagueOfLegends.ChampionModules.Common;
+using Games.LeagueOfLegends.Model;
+using LedDashboardCore;
+using LedDashboardCore.Modules.BasicAnimation;
+
+namespace Games.LeagueOfLegends
+{
+    public abstract class BaseGameModule : LEDModule
```

Comment:
What exactly is this class for? `GameElementModule` is already an abstraction for `ChampionModule` and `ItemModule`.

Reply:
This is an abstraction for `LeagueOfLegendsModule` and `GameElementModule`